pkg/aqua/command.go

```diff
@@ -86,7 +88,7 @@ func (c *command) Exec(imageRef ImageRef) (report ScanReport, err error) {
 	cmd.Stderr = &stderrBuffer
 
 	stdout, err := cmd.Output()
-	if err != nil {
+	if err != nil && cmd.ProcessState.ExitCode() != assurancePolicyCheckFailed {
```

Comment:
Wonder if we should log a warning if the assurance policy check fails, wdyt?

Reply:
Added log message:

{"image_ref":{"Repository":"library/jenkins","Tag":"2.60.3","Digest":"sha256:0de43cde2c4b864a8e4a84bbd9958e47c5d851319f118203303d040b0a74f159"},"level":"warning","msg":"Some assurance policy checks configured in Aqua management console failed. Navigate to Aqua management console for details.","time":"2020-01-02T15:25:20Z"}